decls/postcss.js

```diff
@@ -32,3 +32,9 @@ export type postcss$rule = {
   parent: Object,
   nodes: Array<Object>,
 }
+
+export type postcss$Options = {
+  from?: ?string,
```

Comment:
You shouldn't need both question marks. I think just `from?: string,` should be fine.